dandischema/exceptions.py

```diff
@@ -1,9 +1,22 @@
-class ValidationError(ValueError):
-    """Validation errors were detected.
+from typing import Any, Dict, List
+import jsonschema.exceptions
+
+
+class JsonschemaValidationError(ValueError):
```

Comment:
Let's then may be keep original `ValidationError` as the `super` class for both more specific types of errors?